mne/preprocessing/ieeg/_projection.py

```diff
@@ -16,7 +16,8 @@
 
 @verbose
 def project_sensors_onto_brain(info, trans, subject, subjects_dir=None,
-                               picks=None, n_neighbors=10, verbose=None):
+                               picks=None, n_neighbors=10, copy=False,
```

Comment:
can you make copy = True by default? otherwise some people will trip... sorry to be a pain...

Reply:
Sure, no problem

Reply:
Sure, no problem